doc/mgr/administrator.rst

```diff
@@ -51,7 +51,7 @@ more than ``mon mgr beacon grace`` (default 30s), then it will be replaced
 by a standby.
 
 If you want to pre-empt failover, you can explicitly mark a ceph-mgr
-daemon as failed using ``ceph mgr fail <mgr name>``.
+daemon as failed using ``ceph mgr fail <role>``.
```

Comment:
Another find/replace mishap.

Reply:
It was not accidental. I have made the change purposefully. I have noticed this in mgr and I wanted the reviewer to verify it and approve / disapprove.